docs/api-routes/edge-api-routes.md

```diff
@@ -0,0 +1,115 @@
+---
+description: Edge API Routes enable you to build high performance APIs directly inside your Next.js application.
+---
+
+# Edge API Routes (Beta)
+
+Edge API Routes enable you to build high performance APIs with Next.js. By using the [Edge Runtime](/docs/api-reference/edge-runtime.md), they are faster and more secure than Node.js based API Routes. This performance improvement does come with [constraints](/docs/api-reference/edge-runtime.md#unsupported-apis), like not having access to native Node.js APIs. Instead, Edge API Routes are built on standard Web APIs.
```

Comment:
```suggestion
Edge API Routes enable you to build high performance APIs with Next.js. By using the [Edge Runtime](/docs/api-reference/edge-runtime.md), they are faster and more secure than Node.js-based API Routes. This performance improvement does come with [constraints](/docs/api-reference/edge-runtime.md#unsupported-apis), like not having access to native Node.js APIs. Instead, Edge API Routes are built on standard Web APIs.
```

Reply:
Also, a blanket claim that they are faster and more secure feels a little bold to me - perhaps "are often faster and more secure" would be a little less absolutist?